GoogleDataTransport/GDTCORTests/Lifecycle/GDTCORLifecycleTest.m

```diff
@@ -105,10 +108,8 @@ - (void)DISABLED_testBackgrounding {
       },
       5.0);
 
-  // TODO(#3973): This notification no longer triggers the `isRunningInBackground` flag. Find
-  // another way to test it.
   NSNotificationCenter *notifCenter = [NSNotificationCenter defaultCenter];
-  [notifCenter postNotificationName:kGDTCORApplicationDidEnterBackgroundNotification object:nil];
+  [notifCenter postNotificationName:UIApplicationDidEnterBackgroundNotification object:nil];
```

Comment:
Why change from the abstracted notification to an iOS-specific one?

Reply:
We previously used the abstract one because it triggered each classes' `foreground/background` flags, but now since the flag moved it's set before the abstracted notification is fired. Meaning: the abstract notification no longer triggers the flag to change, only the iOS specific one does.